package.json

```diff
@@ -60,6 +60,7 @@
     "console": "^0.7.2",
     "cookie-universal": "2.2.2",
     "core-js": "3.25.3",
+    "cross-spawn": "^7.0.3",
```

Comment:
Please indicate in the commit message _why_ we need this. It's unclear what issue pulling in a new dependency solves.